frontend/src/core/editor/components/EditorMenu.js

```diff
@@ -60,11 +60,16 @@ export default class EditorMenu extends React.Component<Props> {
                         updateSetting={ props.updateSetting }
                     />
                     <editor.KeyboardShortcuts />
-                    <editor.TranslationLength
-                        entity={ props.entity }
-                        pluralForm={ props.pluralForm }
-                        translation={ props.editor.translation }
-                    />
+                    { (
+                        props.entity.format === 'ftl' ||
+                        typeof(props.editor.translation) !== 'string'
```

Comment:
This is all Flow's fault. That's why it took so long for me to finish this: if I use this simpler solution of putting all types of translations in the same variable (string and AST) then it works, but Flow requires to check types a lot. I tried a bunch of different ways of storing that data in separated variables, but none I tried ended up working, because there are so many moving parts around the editor.